h2o-py/h2o/automl/autoh2o.py

```diff
@@ -15,6 +15,11 @@ class H2OAutoML(object):
 
     :param int nfolds: Number of folds for k-fold cross-validation. Defaults to 5. Use 0 to disable cross-validation; this will also 
       disable Stacked Ensemble (thus decreasing the overall model performance).
+    :param bool balance_classes: Balance training data class counts via over/under-sampling (for imbalanced data).  Defaults to ``false``.  
```

Comment:
Docs say default is `false` but the constructor has it set to `True`

Reply:
Good catch.  Will fix.